newm-server/src/main/kotlin/io/newm/server/database/migration/V53__CreateReleases.kt

```diff
@@ -0,0 +1,90 @@
+package io.newm.server.database.migration
+
+import org.flywaydb.core.api.migration.BaseJavaMigration
+import org.flywaydb.core.api.migration.Context
+import org.jetbrains.exposed.sql.transactions.transaction
+
+class V53__CreateReleases : BaseJavaMigration() {
+    override fun migrate(context: Context?) {
+        transaction {
+            execInBatch(
+                listOf(
+                    """
+                    CREATE TABLE IF NOT EXISTS releases
+                    (
+                        id uuid PRIMARY KEY,
+                        owner_id uuid NOT NULL,
+                        release_type varchar(20) NOT NULL,
+                        title text NOT NULL,
+                        distribution_release_id bigint,
+                        barcode_type integer,
+                        barcode_number text,
+                        release_date DATE,
+                        publication_date DATE,
+                        cover_art_url text,
+                        arweave_cover_art_url text,
+                        has_submitted_for_distribution boolean NOT NULL DEFAULT FALSE,
+                        error_message text,
+                        force_distributed boolean NOT NULL DEFAULT FALSE,
+                        archived boolean NOT NULL DEFAULT FALSE,
+                        created_at timestamp without time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
+                        CONSTRAINT fk_releases_owner_id__id FOREIGN KEY (owner_id) REFERENCES users (id) ON UPDATE NO ACTION ON DELETE NO ACTION
+                    )
+                    """.trimIndent(),
+                    """
+                    ALTER TABLE songs
+                    RENAME COLUMN album TO release_id;
```

Comment:
Are we currently using the `album` to store the `release_id`? Just making sure no side effects of this

Reply:
It's currently doing nothing. The thought was it would contain the album title, but we now store that in the release.title field.